_data/meetups.yml

```
@@ -1429,7 +1429,7 @@
   end_time: 20:00:00 EDT
   url: https://lu.ma/af2n3jwa
 
-- name: Austin.rb - Placeholder
+- name: Austin.rb - Visualizing Problems: From Abstract to Concrete
```

If you have a value with a `:` you need to wrap the whole value in quotes 
```suggestion
- name: "Austin.rb - Visualizing Problems: From Abstract to Concrete"
```

---

done! thanks for the quick reply.